saleor/settings.py

```diff
@@ -498,6 +498,8 @@ def get_host():
 # Slug for channel precreated in Django migrations
 DEFAULT_CHANNEL_SLUG = os.environ.get("DEFAULT_CHANNEL_SLUG", "default-channel")
 
+# Set "True" for cloud instances
+IS_CLOUD_INSTANCE = get_bool_from_env("IS_CLOUD_INSTANCE", False)
```

Comment:
Maybe we should use a name like `ALLOWS_DOMAIN_NAME_CHANGE`? 
@maarcingebala @korycins

Reply:
IS_CLOUD_INSTANCE is already used in related dashboard PR.